### PR TITLE
os/se/sss: Modify printf to vdbg in security_hal_warpper

### PR DESCRIPTION
**Pull request by SeonghoByeon**
- Change the definition of HWRAP_LOG from printf to vdbg

### CODE REVIEW

File: os/se/sss/security_hal_wrapper.c [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
 #define ISP_CHECKBUSY() while (isp_get_status()) {}
 
-#define HWRAP_LOG printf
+#define HWRAP_LOG vdbg
```
Comment (sunghan-chang) [inline os/se/sss/security_hal_wrapper.c]: Why do you use vdbg? This is not enough important to print with CONFIG_DEBUG_VERBOSE enabled.
Please use se-specific debug option.

Comment (SeonghoByeon) [reply os/se/sss/security_hal_wrapper.c]: Thanks for the comment. Made a specific debug option and related code for security hal wrapper layer. Could you review the code if available.
Thanks again!

File: os/se/sss/security_hal_wrapper.c [hunk 2]
```diff
@@ -51,7 +51,12 @@
 
 #define ISP_CHECKBUSY() while (isp_get_status()) {}
 
-#define HWRAP_LOG printf
+#ifdef CONFIG_HAL_WRAPPER_DEBUG
+#define HWRAP_LOG(format, ...) printf(format, ##__VA_ARGS__)
```
Comment (sunghan-chang) [inline os/se/sss/security_hal_wrapper.c]: Please don't use `printf` in kernel. Please use xxdbg like other kernel module.

Comment (SeonghoByeon) [reply os/se/sss/security_hal_wrapper.c]: Thanks for the comment. I will update soon.

**Pull request closed without merging**